modules/programs/thunderbird.nix

```diff
@@ -298,6 +332,41 @@ in {
           };
         });
     };
+    accounts.calendar.accounts = mkOption {
```

Comment:
I prefer to have it the way it is now since it allows the documentation to attribute the options with the correct Nix file.

Reply:
> I prefer to have it the way it is now since it allows the documentation to attribute the options with the correct Nix file.

cant this be solved with a `_file` or something at the top of the module ?